pages/k8s/1.27/components.md

```diff
@@ -57,11 +57,11 @@ These charms are the core components of the Charmed Kubernetes release.
 ## CNI charms
 
 These are the Container Network Interface (CNI) charms supported by Charmed Kubernetes
-(calico is the default CNI and is also included in the core charms table above).
+(Calico is the default CNI and is also included in the core charms table above).
 
 |charm | summary | source | docs | bugs |
 |-|-|-|-|-|
-| [calico](https://charmhub.io/calico) | A robust Software Defined Network from Project Calico | [source](https://github.com/charmed-kubernetes/layer-calico.git) | [docs](https://charmhub.io/calico/docs) | [bugs](https://bugs.launchpad.net/charm-calico) |
+| [calico](https://charmhub.io/calico) | A robust Software Defined Network from Project Calico | [source](https://github.com/charmed-kubernetes/charm-calico.git) | [docs](https://charmhub.io/calico/docs) | [bugs](https://bugs.launchpad.net/charm-calico) |
```

Comment:
the `layer-calico` repo was renamed to `charm-calico`. the former automatically forwards to the latter.  we can change it everywhere since they both go to the same place, but only releases > 1.27+ck2 *technically* pull from the `charm-calico` repo.

they all *effectively* pull from `charm-calico`, but anything < 1.27+ck2 *technically* pulls from `layer-calico`.

i know it's pedantic, but i think it's nice to see where the bits originally came from.  For example, anything < 1.23 pulled from:

https://github.com/juju-solutions/layer-calico

which today gets forwarded to the `charmed-kubernetes` org and then forwarded again to the `charm-calico` repo, It could be nice to know the historical origin for old releases in case you wanted to do something like search for commits to the original repo.

wdyt?  i'm not willing to die on this hill, so i'll defer to you if it's better to be historically vs presently accurate.